project/Deps.sc

```diff
@@ -13,10 +13,13 @@ object Izumi {
     val scalatest = Version.VExpr("V.scalatest")
     val cats = Version.VExpr("V.cats")
     val cats_effect = Version.VExpr("V.cats_effect")
+    val discipline = Version.VExpr("V.discipline")
+    val discipline_scalatest = Version.VExpr("V.discipline_scalatest")
     val zio = Version.VExpr("V.zio")
     val zio_interop_cats = Version.VExpr("V.zio_interop_cats")
     val monix = Version.VExpr("V.monix")
     val monix_bio = Version.VExpr("V.monix_bio")
+    val monix_cats = Version.VExpr("V.monix_cats")
```

Comment:
@vpodlnk This is never used

Reply:
Oh, yup, sorry, I forgot to remove this.

Done ✔️